projects/swimlane/ngx-charts/src/lib/models/legend.model.ts

```diff
@@ -0,0 +1,9 @@
+import { ColorHelper } from '../common/color.helper';
+
+export interface ILegendOptions {
```

Comment:
There's an interface already defined in https://github.com/swimlane/ngx-charts/blob/6d591944a7eaa2942b4b4c39c9d3526d28ceb1b9/projects/swimlane/ngx-charts/src/lib/common/types/legend.model.ts#L3